solver/src/driver.rs

```diff
@@ -74,7 +75,7 @@ impl Driver {
                 )
                 .gas(8_000_000u32.into())
         };
-        settle().call().await.context("settle simulation failed")?;
+        //settle().call().await.context("settle simulation failed")?;
         settle().send().await.context("settle execution failed")?;
```

Comment:
Should the call be commented out?

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2F3o6ZsSRkXifoEJfpMA%2Fgiphy-downsized-medium.gif&token=great_catch" data-gitmeme-token="great_catch"><img src="https://media2.giphy.com/media/3o6ZsSRkXifoEJfpMA/giphy-downsized-medium.gif" title="Created by gitme.me with /great_catch"/></a>